doc/reference/configuration/configuration_reference.rst

```diff
@@ -2174,3 +2174,340 @@ The ``security`` section defines configuration parameters related to various sec
+    megabytes per second it can write to disk. The same can be
+    achieved by splitting :ref:`wal.dir <configuration_reference_wal_dir>` and
+    :ref:`snapshot.dir <configuration_reference_snapshot_dir>`
+    locations and moving snapshots to a separate disk.
+    The limit also affects what
+    :ref:`box.stat.vinyl().regulator <box_introspection-box_stat_vinyl_regulator>`
+    may show for the write rate of dumps to ``.run`` and ``.index`` files.
+
+    |
+    | Type: number
+    | Default: box.NULL
+    | Environment variable: TT_SNAPSHOT_SNAP_IO_RATE_LIMIT
+
+..  _configuration_reference_snapshot_count:
+
+..  confval:: snapshot.count
+
+    The maximum number of snapshots that may exist on the
+    :ref:`snapshot.dir <configuration_reference_snapshot_dir>` directory
+    before the :ref:`checkpoint daemon <configuration_persistence_checkpoint_daemon>` deletes old snapshots.
+    If ``snapshot.count`` equals zero, then the checkpoint daemon
+    does not delete old snapshots.
+
+    Example:
+
+    ..  code-block:: yaml
+
+        snapshot:
+          by:
+            interval: 60
+          count: 10
+
+    In the example, the checkpoint daemon creates a new snapshot each hour until
+    it has created ten snapshots. After that, it deletes the oldest snapshot
+    (and any associated write-ahead-log files) after creating a new one.
+
+    ..  NOTE::
+
+        Snapshots will not be deleted if replication is ongoing and the file has not been relayed to a replica.
+        Therefore, ``snapshot.count`` has no effect unless all replicas are alive.
+
+    |
+    | Type: integer
+    | Default: 2
+    | Environment variable: TT_SNAPSHOT_COUNT
+
+..  _configuration_reference_snapshot_by:
+
+snapshot.by.*
+~~~~~~~~~~~~~
+
+..  _configuration_reference_snapshot_by_interval:
+
+..  confval:: snapshot.by.interval
+
+    The interval in seconds between actions by the :ref:`checkpoint daemon <configuration_persistence_checkpoint_daemon>`.
+    If the option is set to a value greater than zero, and there is
+    activity which causes change to a database, then the checkpoint daemon calls
+    :doc:`box.snapshot() </reference/reference_lua/box_snapshot>` every ``snapshot.by.interval``
+    seconds, creating a new snapshot file each time.
+    If the option is set to zero, the checkpoint daemon is disabled.
+
+    Example:
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/persistence_snapshot/config.yaml
+        :language: yaml
+        :start-at: by:
+        :end-at: interval: 60
+        :dedent:
+
+    In the example, the checkpoint daemon creates a new database snapshot once per minute, if there is activity.
+
+    |
+    | Type: number
+    | Default: 3600
+    | Environment variable: TT_SNAPSHOT_BY_INTERVAL
+
+..  _configuration_reference_snapshot_by_wal_size:
+
+..  confval:: snapshot.by.wal_size
+
+    The threshold for the total size in bytes of all WAL files created since the last snapshot taken.
```

Comment:
Hm. AFAIU we have one WAL file after a last snapshot.

Reply:
I see now: it depends on `wal.max_size`, right? OK so.

(However, I would ask someone from the scaling team to take a glance.)